tools/run_ci.sh

```diff
@@ -33,9 +33,9 @@ bash -x "${SCRIPT_PATH}/run_remote_benchmark.sh"
 ./mc cp results/* qdrant/vector-search-engines-benchmark/results/ci/qdrant/
 
 # Upload to postgres
-
-export SEARCH_RESULTS_FILE=$(ls -ct results/*-search-*.json | head -n 1)
```

Comment:
`-ct` sorts by change time which isn't preserved even on using `scp -p`. So we are using `-t` (modified time). Apparently, they are different. Also, the default sorting of `ls` is based on filename.